### PR TITLE
Added 'vlan config' CLI features to exiting ios_vlans plugin.

### PR DESCRIPTION
**Pull request by PadminiSivaraj**
##### SUMMARY
Added 'vlan config' CLI features to exiting ios_vlans plugin.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
ios_vlans

### CODE REVIEW

**Comment by PadminiSivaraj** [comment]
Hello Sagar,
As discussed, I have dedicated one plugin to deal with two CLIs, so I have added integration test for both of these in a single file, and it is throwing error, could you please shed some light on how to handle this scenario?
Thanks

File: tests/integration/targets/ios_vlans/tests/cli/merged.yaml [hunk 1]
```diff
@@ -60,3 +60,47 @@
   when: ansible_net_version == "15.6(2)T"
   always:
     - ansible.builtin.include_tasks: _remove_config.yaml
+
+- block:
```
Comment (KB-perByte) [inline tests/integration/targets/ios_vlans/tests/cli/merged.yaml:64]: @PadminiSivaraj  the block is a duplicate, the block present can be reused. 
ref: sanity https://github.com/ansible-collections/cisco.ios/actions/runs/6429392546/job/17460329897?pr=917#step:10:283

File: tests/unit/modules/network/ios/test_ios_vlans.py [hunk 2]
```diff
@@ -372,12 +483,42 @@ def test_ios_vlans_overridden_idempotent(self):
...
         self.execute_module(changed=False, commands=[], sort=True)
 
-    def test_ios_delete_vlans_config(self):
+    def test_ios_delete_vlans(self):
         set_module_args(dict(config=[dict(vlan_id=150)], state="deleted"))
         result = self.execute_module(changed=True)
         commands = ["no vlan 150"]
         self.assertEqual(result["commands"], commands)
 
+    def test_ios_delete_vlans_config(self):
```
Comment (KB-perByte) [inline tests/unit/modules/network/ios/test_ios_vlans.py:492]: ref https://github.com/ansible-collections/cisco.ios/actions/runs/6429392546/job/17460328093?pr=917#step:10:793 unit test failure

File: tests/unit/modules/network/ios/test_ios_vlans.py [hunk 3]
```diff
@@ -648,3 +851,52 @@ def test_ios_vlans_gathered(self):
 
         self.maxDiff = None
         self.assertEqual(result["gathered"], gathered)
+
+    def test_ios_vlans_config_gathered(self):
+        set_module_args(dict(configuration=True, state="gathered"))
```
Comment (KB-perByte) [inline tests/unit/modules/network/ios/test_ios_vlans.py:856]: This is generating empty config. Please pass a config block

**Review by KB-perByte — CHANGES_REQUESTED** [review]
There are multiple unit test failures that needs to be fixed

File: tests/integration/targets/ios_vlans/tests/cli/merged.yaml (hunk 1 above)
Comment (PadminiSivaraj) [reply tests/integration/targets/ios_vlans/tests/cli/merged.yaml]: Thanks for the review, Sagar. Fixed.

File: tests/unit/modules/network/ios/test_ios_vlans.py (hunk 2 above)
Comment (PadminiSivaraj) [reply tests/unit/modules/network/ios/test_ios_vlans.py]: Thanks for the review, Sagar. Fixed.

File: tests/unit/modules/network/ios/test_ios_vlans.py (hunk 3 above)
Comment (PadminiSivaraj) [reply tests/unit/modules/network/ios/test_ios_vlans.py]: Thanks for the review, Sagar. Fixed.

**Comment by PadminiSivaraj** [comment]
> There are multiple unit test failures that needs to be fixed

Thanks for the review, Sagar. Fixed the failing checks.

**Pull request merged**